docs/website/docs/reference/performance.md

```diff
@@ -249,6 +250,17 @@ The default is to not parallelize normalization and to perform it in the main pr
 Normalization is CPU bound and can easily saturate all your cores. Never allow `dlt` to use all cores on your local machine.
 :::
 
+:::caution
+The default method of spawning a process pool on Linux is **fork**. If you are using threads in your code (or libraries that use threads),
```

Comment:
should we add a link to some further explanation of this in the python docs maybe?

Reply:
good point! could you propose a link? then I'll add it